package/src/utils.ts

```diff
@@ -37,24 +39,24 @@ export const toMaxDecimalsRound = (n: number | string, decimals: number) => {
     return !isFinite(nRound) ? 0 : nRound
 }
 
-const ordredMagnitudes = [
-    { magnitude: 'seconds', magnitudeShort: 's', factor: 1 },
-    { magnitude: 'minutes', magnitudeShort: 'm', factor: 60 },
-    { magnitude: 'hours', magnitudeShort: 'h', factor: 3600 },
-    { magnitude: 'days', magnitudeShort: 'd', factor: 86400 }
-]
+const orderedMagnitudes = [
+    { magnitude: i18next.t('timeMagnitudes.second'), magnitudeShort: 's', factor: 1 },
+    { magnitude: i18next.t('timeMagnitudes.minute'), magnitudeShort: 'm', factor: 60 },
+    { magnitude: i18next.t('timeMagnitudes.hour'), magnitudeShort: 'h', factor: 3600 },
+    { magnitude: i18next.t('timeMagnitudes.day'), magnitudeShort: 'd', factor: 86400 }
```

Comment:
The `magnitudeShort` values here won't change as they are not translated. This will result in the function below `formatSeconds` returning a format which non English speakers don't understand.

Reply:
This was considered fine by Salah IIRC.